BLE Indoor Positioning/src/test/java/com/nexenio/bleindoorpositioning/location/LocationUtilTest.java

```diff
@@ -0,0 +1,66 @@
+package com.nexenio.bleindoorpositioning.location;
+
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Created by leon on 29.03.18.
+ */
+public class LocationUtilTest {
+
+    @Test
+    public void meanLocation_multipleLocations_correctMeanLocation() throws Exception {
+        List<Location> locationList = new ArrayList<Location>() {{
+            add(LocationTest.NEW_YORK_CITY);
+            add(LocationTest.BERLIN);
+        }};
+        Location expectedLocation = new Location(46.61639095, -30.3005094);
```

Comment:
Also add one more precise example, e.g. the edges of the soccer field and the center point.